ansible/group_vars/all/kolla

```diff
@@ -321,6 +323,7 @@ kolla_enable_neutron: "yes"
 kolla_enable_neutron_provider_networks: "no"
 kolla_enable_nova: "yes"
 kolla_enable_osprofiler: "no"
+kolla_enable_prometheus: "no"
```

Comment:
Are we playing it safe?

Reply:
We'll turn it back on in the kayobe-config :)